girder/web_client/grunt_tasks/build.js

```diff
@@ -282,6 +282,12 @@ module.exports = function (grunt) {
             module: {
                 rules: [babelRule]
             },
+            resolve: {
+                alias: {
+                    // This is deprecated, but kept in place for compatibility
+                    'girder': '@girder/core'
```

Comment:
Does this mean this is not actually a breaking change?

Reply:
Not for downstream plugins.

Reply:
Sorry, you're saying it *is* breaking for downstream plugins, or it's *not*?

Reply:
However, as noted, we might want to intentionally make this a breaking change, if we decide that the 2->3 release is a more opportune time to require that downstreams start using the 'correct' core package name (since they're already changing plugins).